File: python-sdk/fyipe_sdk/fyipe_sdk/fyipeTransport.py
import requests

class FyipeTransport:
    
    def __init__(self, apiUrl):
        # set up the api transporter
        self.apiUrl = apiUrl
    
    def sendErrorEventToServer(self, event):
        response = self._makeApiRequest_(event)
        return response

    def _makeApiRequest_(self, body):
        # make api request and return response
        response = requests.post(self.apiUrl, json=body)
        return response.json()
    

File: ruby-sdk/fyipe/lib/fyipe/fyipeTransport.rb
require 'httparty'

class FyipeTransport
    include HTTParty

    def initialize( apiUrl)
        # set up the api transporter
        @apiUrl = apiUrl
    end
    
    def sendErrorEventToServer(event)
        response = makeApiRequest(event)
        return response
    end

    private
    def makeApiRequest(body)
        # make api request and return response
        params = { body: body }

        response = self.class.post(@apiUrl, params).parsed_response
        return response
    end
end